lib/suites/LinkedDataSignature.js

```diff
@@ -297,9 +310,21 @@ module.exports = class LinkedDataSignature {
         proof
       }));
 
+    // Check that the proofPurpose is valid
+    const {purpose} = options;
+    if(purpose) {
+      const {expanded, purpose: {proofPurposeHandler}} = options;
+      const result = await proofPurposeHandler.verify(
```

Comment:
related to other comments, we do want to pass proofs to verify in expanded form?